src/orange/OrangeTrackView.hh

```diff
@@ -918,8 +954,11 @@ CELER_FUNCTION real_type OrangeTrackView::find_safety()
     auto lsa = this->make_lsa();
 
     CELER_ASSERT(lsa.universe() == UniverseId{0});
-    auto tracker = this->make_tracker(lsa.universe());
-    return tracker.safety(lsa.pos(), lsa.vol());
+
+    TrackerVisitor visit_tracker{params_};
+    return visit_tracker(
```

Comment:
I wonder if it's worth having a `template<class F> auto visit_tracker(F&&, UniverseId) const;` (and same for `visit_transform`)?

Reply:
You mean something like:

```
template<class F> auto OrangeTrackView::visit_tracker(F&& f, UniverseId uid)
{
        TrackerVisitor visit{params_};
        return visit(f, uid);
}
```
?

Reply:
@sethrj, I tried the code above, but I am getting type deduction errors, despite this the same API as what is done within TrackerVisit. The rest of  this PR is good to go.

Reply:
Thanks for trying that out. We can simplify the code in the future if needed.